src/components/shared/DataTable/DataTable.js

```diff
@@ -11,16 +11,10 @@ import { useSticky } from 'react-table-sticky'
 import { Box, Text } from 'grommet'
 import { useIntersectObserver } from 'hooks/useIntersectObserver'
 import { BoxBlock } from 'components/shared/BoxBlock'
-import {
```

Comment:
Please don't use relative imports here

Reply:
In this case, we need to re-do the merged PR https://github.com/AlexsLemonade/refinebio-web/pull/145 as well. Then I'll create a new PR to adjust all the relative imports, please let me know!

Reply:
🗒️  We'll use the following pattern for module imports:
- use the absolute path (`src` as the base url) for shared or page components
- use the relative path for nested components (child components that belong to that parent module and only used in it. e.g., for `ProcessingInformationCell` > `ModalContent`,  use `./ModalContent` since this modal content is specific to the processing information cell and not shared elsewhere)